src/main.js

```diff
@@ -63,6 +64,12 @@ const app = createApp(App)
 app.use(router)
 app.use(i18n)
 app.use(store)
+app.use(VueSocialSharing, {
+    // defining service-qrcode as a network for the sharing library
+    networks: {
+        qrcode: `${API_SERVICES_BASE_URL}qrcode/generate?url=@url`,
+    },
```

Comment:
This seems not to work correctly, I tested the qrcode and it had an invalid short url. In my test the share menu had https://sys-s.dev.bgdi.ch/991e0ea1ec while the qrcode used https://sys-map.dev.bgdi.ch/api/qrcode/generate?url=https%3A%2F%2Fsys-s.dev.bgdi.ch%2F991e0ea1ecrl ! Which indeed did not exists in the  DB backend.

Reply:
that's weird that it could slip by testing... We are specifically testing the URL that is opened by the link, and yeah there's this extra "rl" at the end that is coming from the configuration here : https://github.com/geoadmin/web-mapviewer/blob/86fa42572232b2b21219bfd2e4232a5a4ea205ad/src/modules/menu/components/share/MenuShareSocialNetworks.vue#L54-L58

I'm guessing the doc wasn't updated but now we should declare it `...?url=@u`, it removes this extra `rl` if I do so.
But then it should have been picked by https://github.com/geoadmin/web-mapviewer/blob/86fa42572232b2b21219bfd2e4232a5a4ea205ad/tests/e2e-cypress/integration/shareShortLink.cy.js#L57-L63
I'll look a bit into it, but that's confusing 😕